hazelcast/src/main/java/com/hazelcast/jet/config/JobConfig.java

```diff
@@ -1390,6 +1391,15 @@ public JobConfig setTimeoutMillis(long timeoutMillis) {
         return this;
     }
 
+    /**
+     * Applies the specified changes to this configuration.
+     *
+     * @since 5.3
+     */
+    public JobConfig apply(DeltaJobConfig deltaConfig) {
```

Comment:
I am not sure if we want to expose this method in the public API. It is not useful for the user.